var/spack/repos/builtin/packages/sst-macro/package.py

```diff
@@ -90,6 +93,9 @@ def configure_args(self):
             if '+skeletonizer' in spec:
                 args.append('--with-clang=' + spec['llvm'].prefix)
 
+        if '+core' in spec:
```

Comment:
what happens with `~core`?  I'm not familiar with how these packages are related so I can't say what to do with this one.

Reply:
Core is optional substitute for the PDES (Parallel discreet event simulation) engine, I agree that the naming is confusing.

Reply:
oh.